homeassistant/components/isy994/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Universal Devices ISY/IoX",
   "integration_type": "hub",
   "documentation": "https://www.home-assistant.io/integrations/isy994",
-  "requirements": ["pyisy==3.1.5"],
+  "requirements": ["pyisy==3.1.6"],
   "codeowners": ["@bdraco", "@shbatm"],
```

Comment:
There is a lot going on this PR. Can you move the lib bump to another PR?

Reply:
I will split it out, but it will need to be merged first before this PR and before you test this. I made a mistake in `3.1.5` when I added the filtering code--using `3.1.5` won't break anything existing, but it will break state updates with these changes.

I am going to bump again because someone (::cough::) sent me down a rabbit-hole this morning: https://github.com/automicus/PyISY/pull/369

Reply:
That looks like a rabbit hole. Thanks for trying. I'll get the lib bump merged as soon as I see it

Reply:
Having issues with github right now